autoscale-core/src/main/java/com/hpe/caf/autoscale/core/ScalerThread.java

```diff
@@ -46,6 +46,8 @@
 
     private final Governor governor;
     private final ResourceMonitoringConfiguration resourceConfig;
+    
+    long firstAttempt;
```

Comment:
- Are you intentionally granting package access to this instance variable?
- It seems like a poor name.  It sounds like a flag but it is actually a `long`, so what is it?